src/servant/tt_001/F84.c

```diff
@@ -390,13 +487,13 @@ void func_us_80171864(Entity* self) {
         } else if (D_us_801737CC < 8) {
             self->ext.ghost.unk86++;
             if (self->ext.ghost.unk86 ==
-                (D_us_80170508[((D_us_80173810.level / 10) * 3)] - 0x1E)) {
+                (D_us_80170508[D_us_80173810.level / 10][0] - 0x1E)) {
                 self->ext.ghost.unk92 = func_us_80171568(self, 0);
             } else if (self->ext.ghost.unk86 >
-                       D_us_80170508[((D_us_80173810.level / 10) * 3)]) {
+                       D_us_80170508[D_us_80173810.level / 10][0]) {
                 self->ext.ghost.unk86 = 0;
                 g_api.func_8011A3AC(
-                    self, D_us_8017050C[((D_us_80173810.level / 10) * 3)], 1,
+                    self, D_us_8017050C[D_us_80173810.level / 10][0], 1,
```

Comment:
These are different memory addresses but the same pattern as my func, likely the same struct. I've updated these for consistency.

My guess is that on PSP side these would also access [2]